src/ol/size.js

```diff
@@ -19,3 +39,40 @@ ol.Size;
 ol.size.equals = function(a, b) {
   return a[0] == b[0] && a[1] == b[1];
 };
+
+
+/**
+ * Returns a size scaled by a ratio. The result will be an array of integers.
+ * @param {ol.Size} size Size.
+ * @param {number} ratio Ratio.
+ * @param {ol.Size=} opt_size Optional reusable size array.
+ * @return {ol.Size}
+ */
+ol.size.scale = function(size, ratio, opt_size) {
+  if (!goog.isDef(opt_size)) {
+    opt_size = [0, 0];
+  }
+  opt_size[0] = (size[0] * ratio + 0.5) | 0;
+  opt_size[1] = (size[1] * ratio + 0.5) | 0;
+  return opt_size;
+};
+
+
+/**
+ * @param {number|ol.Size} size Width and height.
+ * @param {ol.Size=} opt_size Optional reusable size array.
+ * @return {ol.Size} Size.
+ */
+ol.size.toSize = function(size, opt_size) {
```

Comment:
`ol.size.toSize` is mentioned in the release notes, so it should be `@api`. And it should have a description. And to be fair with our users I think it should even be `@api stable`, because `getTileSize` is stable.